### PR TITLE
Style fixes (code quality)

### PR DESCRIPTION
**Pull request by ruudboon**
Hello!

* Type: code quality
Some styling fixes and interface mismatch fixes

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [x] I have checked that another pull request for this purpose does not exist
- [ ] I wrote some tests for this PR
- [x] I have updated the relevant CHANGELOG
- [ ] I have created a PR for the [documentation](https://github.com/phalcon/docs) about this change

-- 

- Removed double use use Phalcon\Events\ManagerInterface as EventsManagerInterface
- Interfaces cannot have final
- Typo in inline class
- Interface corrections for handle()
- Styling fix

### CODE REVIEW

File: phalcon/Application/AbstractApplication.zep [hunk 1]
```diff
@@ -94,7 +94,7 @@ abstract class AbstractApplication extends Injectable implements EventsAwareInte
     /**
      * Handles a request
      */
-    abstract public function handle();
+    abstract public function handle(string! uri) -> <ResponseInterface> | bool;
```
Comment (sergeyklay) [inline phalcon/Application/AbstractApplication.zep]: This should be reflected in the change log

File: phalcon/Cli/Console.zep [hunk 2]
```diff
@@ -34,7 +34,7 @@ class Console extends AbstractApplication
     /**
      * Handle the whole command-line tasks
      */
-    public function handle(array arguments = null)
+    public function handle(string! arguments) -> <ResponseInterface> | bool
```
Comment (sergeyklay) [inline phalcon/Cli/Console.zep]: as well as this and any other API changes

File: phalcon/Mvc/Application.zep [hunk 3]
```diff
@@ -79,13 +79,17 @@ class Application extends AbstractApplication
     /**
      * Handles a MVC request
      */
-    public function handle(string! uri) -> <ResponseInterface> | bool
+    public function handle(var! uri)
```
Comment (sergeyklay) [inline phalcon/Mvc/Application.zep]: But why? Previously users was able to use this like:
```php
    $application = new Application($di);

    echo $application->handle()
        ->getContent();
```

What approach do you suggest? It is breaks BC (again). Anyway don't forget to update docs, as well as change log please.

File: phalcon/Mvc/Application.zep [hunk 4]
```diff
@@ -79,13 +79,17 @@ class Application extends AbstractApplication
...
     {
         var container, eventsManager, router, dispatcher, response, view,
             module, moduleObject, moduleName, className, path, implicitView,
             returnedResponse, controller, possibleResponse, renderStatus,
             matchedRoute, match;
 
+        if unlikely typeof uri != "string" {
+            throw new Exception("Argument should be a string");
```
Comment (sergeyklay) [inline phalcon/Mvc/Application.zep]: Better use `handle(string! uri)`

File: phalcon/Cli/Console.zep [hunk 5]
```diff
@@ -34,13 +34,17 @@ class Console extends AbstractApplication
...
-    public function handle(array arguments = null)
+    public function handle(var! arguments)
     {
         var className, container, dispatcher, eventsManager, module, moduleName,
             moduleObject, modules, path, router, task;
 
         let container = this->container;
 
+        if unlikely typeof arguments != "array" {
+            throw new Exception("Argument should be an array");
```
Comment (sergeyklay) [inline phalcon/Cli/Console.zep]: `public function handle(array arguments = null)` solves the task

File: phalcon/Mvc/Application.zep (hunk 4 above)
Comment (ruudboon) [reply phalcon/Mvc/Application.zep]: Ok will update the exception. We need to change this.
https://github.com/phalcon/cphalcon/blob/a48b31967a1ce27c10cbed61bee42e0c90addc86/phalcon/Application/AbstractApplication.zep#L97
https://github.com/phalcon/cphalcon/blob/a48b31967a1ce27c10cbed61bee42e0c90addc86/phalcon/Mvc/Application.zep#L82
https://github.com/phalcon/cphalcon/blob/a48b31967a1ce27c10cbed61bee42e0c90addc86/phalcon/Cli/Console.zep#L37

Comment (sergeyklay) [reply phalcon/Mvc/Application.zep]: I propose just remove `  abstract public function handle(); `. What this solves? It is not an interface, just restriction for users, isn't?

File: phalcon/Application/AbstractApplication.zep [hunk 6]
```diff
@@ -94,7 +94,7 @@ abstract class AbstractApplication extends Injectable implements EventsAwareInte
     /**
      * Handles a request
      */
-    abstract public function handle();
+    abstract public function handle(var! uri);
```
Comment (niden) [inline phalcon/Application/AbstractApplication.zep]: Lets remove this from here

File: phalcon/Cli/Console.zep (hunk 5 above)
Comment (niden) [reply phalcon/Cli/Console.zep]: And do what Serghei says - remove the exception checking also

File: phalcon/Mvc/Application.zep (hunk 3 above)
Comment (niden) [reply phalcon/Mvc/Application.zep]: And we can leave this the way it was.

**Comment by niden** [comment]
@ruudboon Some comments above that would most likely solve this.

**Review by Jeckerson — DISMISSED** [review]

**Review by sergeyklay — DISMISSED** [review]

**Review by niden — APPROVED** [review]

**Pull request merged**